### ISSUE TITLE
Status monitor timestamps are an hour too fast

### ISSUE THREAD

**Issue by melak** [grke/burp#719]
May or may not be related to #493 and friends, I opted for a new issue.

Server and client both 2.2.2 on Linux. The times reported by the status monitor are one hour in the future. Not sure what to make of this - I am in UTC+0200, can't really figure out where UTC+0300 could possibly come from.

```
burpClient# /opt/burp/sbin/burp -c /etc/opt/burp/burpClient.conf -a S -C burpClient | head -10

 burp status                                          2018-05-29 12:33:03 +0200

   Client: burpClient
   Status: idle
   Backup list: 0000217 2018-05-29 13:30:01 +0200 (current)
                0000216 2018-05-29 13:20:01 +0200
                0000215 2018-05-29 13:10:02 +0200
                0000214 2018-05-29 13:00:01 +0200
                0000213 2018-05-29 12:50:02 +0200

burpClient# /opt/burp/sbin/burp -Q -c /etc/opt/burp/burpClient.conf -a l | tail -6
Backup: 0000212 2018-05-29 11:40:01 +0200
Backup: 0000213 2018-05-29 11:50:02 +0200
Backup: 0000214 2018-05-29 12:00:01 +0200
Backup: 0000215 2018-05-29 12:10:02 +0200
Backup: 0000216 2018-05-29 12:20:01 +0200
Backup: 0000217 2018-05-29 12:30:01 +0200

burpServer# ls -latr | tail
drwxr-xr-x   3 burp burp  4096 May 29 11:30 0000211 2018-05-29 11:30:01 +0200
drwxr-xr-x   3 burp burp  4096 May 29 11:40 0000212 2018-05-29 11:40:01 +0200
drwxr-xr-x   3 burp burp  4096 May 29 11:50 0000213 2018-05-29 11:50:02 +0200
drwxr-xr-x   3 burp burp  4096 May 29 12:01 0000214 2018-05-29 12:00:01 +0200
drwxr-xr-x   3 burp burp  4096 May 29 12:10 0000215 2018-05-29 12:10:02 +0200
drwxr-xr-x   3 burp burp  4096 May 29 12:20 0000216 2018-05-29 12:20:01 +0200
lrwxrwxrwx   1 burp burp    33 May 29 12:30 current -> 0000217 2018-05-29 12:30:01 +0200
-rw-r--r--   1 burp burp  6493 May 29 12:30 dfiles
drwxr-xr-x   3 burp burp  4096 May 29 12:30 0000217 2018-05-29 12:30:01 +0200
```

**Comment by grke** [comment]
Hello,

It seems more likely to be #714, which is (probably) fixed in the release I am doing today (2.2.4).

**Comment by melak** [comment]
Yes, it seems to be fixed in 2.2.4. Cheers!
